src/App.js

```diff
@@ -29,6 +32,8 @@ export const App = () => {
   useGetCategories();
   useGetVideos();
   useGetHistory();
+  useGetWatchLater();
+  useGetLikedVideos();
   // useSignup();
```

Comment:
remove commented code if not going to use it in future

Reply:
Will do it after I'm done with the playlist feature.